src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -5004,6 +5004,12 @@ int wm_vuldet_collect_agent_software(scan_agent *agent, sqlite3 *db, scan_ctx_t*
                 architecture = agent->arch;
             }
 
+            char* win_architecture = NULL;
```

Comment:
Although this translation only apply to windows agents, what we are doing is translate to the NVD architecture format so, I would call this variable `nvd_architecture`.

Reply:
Fixed!